packages/dynamite/dynamite/lib/src/builder/serializer.dart

```diff
@@ -5,15 +5,22 @@ List<Spec> buildSerializer(final State state) => [
       const Code('// coverage:ignore-start\n'),
       Field((final b) {
         b
-          ..docs.add('''
+          ..docs.add(r'''
 /// Serializer for all values in this library.
 /// 
 /// Serializes values into the `built_value` wire format.
-/// See: [jsonSerializers] for serializing into json.''')
+/// See: [$jsonSerializers] for serializing into json.''')
           ..annotations.add(refer('visibleForTesting'))
           ..modifier = FieldModifier.final$
           ..type = refer('Serializers')
-          ..name = 'serializers';
+          ..name = r'$serializers'
+          ..assignment = const Code(r'_$serializers');
+      }),
+      Field((final b) {
+        b
+          ..modifier = FieldModifier.final$
+          ..type = refer('Serializers')
+          ..name = r'_$serializers';
```

Comment:
Why have a public and a private variable that are the same? One public one with the $ prefix is enough

Reply:
Nope it isn't:
In the method body of an operation we need private variables.
Imagine an operation with a parameter called `jsonSerializers`:

```dart
  DynamiteRawResponse<JsonObject, void> $getRaw({
    String? jsonSerializers,
  }) {
    final _parameters = <String, dynamic>{};
    const _headers = <String, String>{
      'Accept': 'application/json',
    };

    final $jsonSerializers = jsonSerializers.serialize( // need a private serializer here
      contentString,
      specifiedType: const FullType(ContentString, [
        FullType(BuiltMap, [FullType(String), FullType(JsonObject)]),
      ]),
    );
    _parameters['json-serializers'] = $jsonSerializers;
    
    // ...
  }
```

Reply:
But something like
```dart
    final _$jsonSerializers = $jsonSerializers.serialize( // need a private serializer here
      jsonSerializers,
      specifiedType: const FullType(String),
    );
```
should work.
The serialized name just needs to be prefixed with underscore.

Reply:
but then a parameter called `headers` or `path` (we already faced the later one once) would conflict with our already private variables.

Yes we can prefix them with `_$` but I don't know if we should rework such a major thing now.

Reply:
Alright then